pkgs/by-name/wl/wlr-layout-ui/package.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "wlr-layout-ui";
+  version = "1.4.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "fdev31";
+    repo = "wlr-layout-ui";
+    rev = "${version}";
+    hash = "sha256-YGETpbgY/KIw680H/4J6TZMaHNP5mNhEuDyYvD5BeIE=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pyglet
+    tomli
+    tomli-w
+  ];
+
+  meta = with lib; {
+    description = "A simple GUI to setup the screens layout on wlroots based systems.";
+    homepage = "https://github.com/fdev31/wlr-layout-ui/";
+    changelog = "https://github.com/fdev31/wlr-layout-ui/releases/tag/${src.rev}";
+    maintainers = with maintainers; [ bnlrnz ];
+    license = licenses.unfree;
```

Comment:
I'd suggest you open an issue and ask the author which license they tend to use. L

Reply:
Also ask the author. If the number of contributors is limited, they can decide how to distribute legacy code (without violating the licenses of others).